### DISCUSSION TITLE
tumblthree  online-status

### DISCUSSION THREAD

**Discussion by accafilm** [TumblThreeApp/TumblThree#518] [Q&A (Question & Answer)]
Hello! in my situation, there are several tumblr pages which are labeled "offline", althoug they are "active" while opended in firefox e.g.the version of the tumblThree is 2.12.0.438  operating system WIN 10.

thank You in advance for hints to help understanding!!

**Comment by MSWallack** [comment]
Sometimes Tumblr blogs get marked as offline if they can't be accessed at the exact moment the app tries to access them (or, sometimes, when you're not logged in properly). If the blog is, in fact, active, usually right-clicking on the blog entry in the app and then selecting "Checking Blog Status" (or something like that; I'm not in front of the app at the moment), will cause the app to essentially refresh whether it thinks the blog is offline or not.
